paasta_tools/deployd/watchers.py

```diff
@@ -303,10 +312,9 @@ def watch_new_folder(self, event):
     def process_default(self, event):
         self.log.debug(event)
         self.watch_new_folder(event)
-        event = self.filter_event(event)
-        if event:
+        service_name = self.get_service_name_from_event(event)
+        if service_name:
             self.log.info("Change of {} in {}".format(event.name, event.path))
```

Comment:
Can you add "scheduling a bounce for $service?

Reply:
I think this log line is okay. Technically we haven't decided what service instances to bounce yet. That happens in `bounce_service`. So this just logs that we have detected a change in a relevant file.